osu.Game/Screens/SelectV2/PanelSetBackground.cs

```diff
@@ -133,7 +145,6 @@ private void loadContentIfRequired()
 
             LoadComponentAsync(new PanelBeatmapBackground(working)
             {
-                Depth = float.MaxValue,
```

Comment:
Depth specification removed here so the background is layered between the placeholder gray box and the dim layers.